src/components/AccountSettingsV2.tsx

```diff
@@ -0,0 +1,279 @@
+import React, { useEffect, useRef, useState } from 'react'
+import { Button, Link } from '@mui/material'
+import { UserBundle, UserProfile } from 'synapse-react-client/dist/utils/synapseTypes'
+import { SynapseClient, SynapseConstants } from 'synapse-react-client'
+import { useSynapseContext } from 'synapse-react-client/dist/utils/SynapseContext'
+import { displayToast } from 'synapse-react-client/dist/containers/ToastMessage'
+import { Form, FormControl, FormGroup, FormLabel } from 'react-bootstrap'
+import { ChangePassword } from './ChangePassword'
+import IconSvg from 'synapse-react-client/dist/containers/IconSvg'
+import { useHistory } from 'react-router-dom'
+import { ORCiDButton } from './ORCiDButton'
+
+export const AccountSettings2 = () => {
+  const { accessToken } = useSynapseContext()
+  const [ userProfile, setUserProfile ] = useState<UserProfile>()
+  const [ firstName, setFirstName ] = useState<string>()
+  const [ lastName, setLastName ] = useState<string>()
+  const [ position, setPosition ] = useState<string>()
+  const [ company, setCompany ] = useState<string>()
+  const [ location, setLocation ] = useState<string >()
+  const [ url, setUrl ] = useState<string>()
+  const [ industry, setIndustry ] = useState<string>()
+  const [ primaryEmail, setPrimaryEmail ] = useState<string>()
+  const [ username, setUsername ] = useState<string>()
+  const [ changeInForm, setChangeInForm] = useState(false)
+  const [ orcid, setOrcid ] = useState<string>()
+  const [ verified, setVerified ] = useState<boolean>()
+  const [ isCertified, setIsCertified ] = useState<boolean>()
+  const history = useHistory()
+
+  const profileInformationRef = useRef<HTMLDivElement>(null)
+  const changePasswordRef = useRef<HTMLDivElement>(null)
+  const trustCredentialRef= useRef<HTMLDivElement>(null)
+  const personalAccessTokenRef = useRef<HTMLDivElement>(null)
+
+  // const scrollEffect = (targetRef:HTMLDivElement) => {
+  //   targetRef.current.scrollIntoView({
+  //     behaviour: 'smooth',
+  //     block: 'start'
+  //   })
+  // }
+  const handleChangesFn = (val: string) => {
+    history.push(`/authenticated/${val}`)
+  }
+  const markFormDirty = () => setChangeInForm(true)
```

Comment:
When any sort of text is written in the form below this changes the disabled button. However, when the changed value is reverted back the button is still enabled.
There needs to be a way to disable the button when value has not changed.